source/speech/speech.py

```diff
@@ -969,6 +972,23 @@ def speak(  # noqa: C901
 	import speechViewer
 	if speechViewer.isActive:
 		speechViewer.appendSpeechSequence(speechSequence)
+	if config.conf['braille']['mode'] == braille.BrailleMode.SPEECH_EMULATION.value:

```

Comment:
please factor out this block to a helper function, note the C901 warning for the function